docs/changelog/3502.feature.rst

```diff
@@ -0,0 +1 @@
+Add ranges to generative environments such as py3{10-13}. - by :user:`mimre25`
```

Comment:
`Add support for number ranges in generative environments`, and also add `, more details here` where here is a link to the documentation.